src/ol_infrastructure/infrastructure/elasticsearch/__main__.py

```diff
@@ -125,16 +127,16 @@
     ingress=[
         ec2.SecurityGroupIngressArgs(
             protocol="tcp",
-            from_port=9200,
-            to_port=9200,
+            from_port=9200,  # noqa: WPS432
```

Comment:
For these kinds of magic numbers I actually created a module (I think called magic_numbers) under utils to hold those values and then import and use the descriptive names in these locations. That's preferable to sprinkling `noqa` statements everywhere we use a number, at least for well-known ports, etc.

Reply:
That util is in the configuration repo, so leaving this as is for now.